protobuf/common/common.pb.go

```diff
@@ -3,8 +3,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.30.0
-// 	protoc        v3.20.3
+// 	protoc-gen-go v1.28.1
+// 	protoc        v3.19.6
```

Comment:
Please update your local installation: our Generate GHA uses `v1.30.0` and `v3.20.3`.